docs/src/components/Accordion/Accordion.stories.mdx

```diff
@@ -1,69 +1,156 @@
+
+## Demo
+
+### Basic accordion
+
+#### light
 
-```javascript
-import {Accordion, AccordionListType} from 'dooboo-ui';
+<AccordionStory theme="light" data={sampleData} />
 
-const Default = (): React.ReactElement => {
-  const data: AccordionListType = [
-    {
-      title: 'Default-title-01',
-      bodies: ['Default body01', 'Default body02'],
+#### dark
+
+<AccordionStory theme="dark" data={sampleData} />
+
+### Animation Setting
+
+This component supports drop down toggle with animation. Animation actions are based [Animated](https://reactnative.dev/docs/animated) library for React-Native, You can set an animation effect starts and duration when this component is mounted.
+
+<AnimationAccordion data={sampleData} />
+
+## Customization
+
+### Styled accordion
+
+Customize `titleContainer` or `bodyContainer` style with style object.
+
+```tsx
+<Accordion
+  styles={{
+    titleContainer: {
+      backgroundColor: 'gray',
+      alignItems: 'center',
+      height: 60,
     },
-    {
-      title: 'Default-title-02',
-      bodies: ['Default body01', 'Default body02'],
+    bodyContainer: {
+      backgroundColor: 'lightgray',
+      borderTopWidth: 0,
+      borderWidth: 0.5,
+      borderColor: 'gray',
+      justifyContent: 'center',
+      height: 20,
     },
-    {
-      title: 'Default-title-03',
-      bodies: ['Default body01', 'Default body02'],
+  }}
+  {...props}
+/>
+```
+
+<AccordionStory
+  data={sampleData}
+  styles={{
+    titleContainer: {
+      backgroundColor: 'gray',
+      alignItems: 'center',
+      height: 60,
+    },
+    bodyContainer: {
+      backgroundColor: 'lightgray',
+      borderTopWidth: 0,
+      borderWidth: 0.5,
+      borderColor: 'gray',
+      justifyContent: 'center',
+      height: 20,
     },
-  ];
+  }}
+/>
+
+### Using render function
+
+It supports seperate render function for Accordion. You can customize the look of each component with the `renderTitle` or `renderBody` function.
+
+```tsx
+import {Icon} from 'dooboo-ui';
+import styled from '@emotion/native';
 
-  return (
+const Container = styled.View`
+  padding-left: 20px;
+  align-items: center;
+  flex-direction: row;
+`;
+
+const Title = styled.Text`
+  margin-left: 20px;
+  color: white;
+  font-size: 18px;
+`;
+
+const Body = styled.Text`
+  flex: 1;
+  width: 80px;
+  font-weight: bold;
+`;
+
+<Accordion
+  renderTitle={(item) => (
     <Container>
-      <Accordion
-        data={data}
-        shouldAnimate={true}
-        collapseOnStart={true}
-        animDuration={400}
-        activeOpacity={1}
-        toggleElement={<Arrow />}
-      />
+      <Icon name="search-light" color="white" />
+      <Title>{item}</Title>
     </Container>
-  );
-};
+  )}
+  renderBody={(item) => (
+    <Container>
+      <Body numberOfLines={1}>{item.toUpperCase()}</Body>
+    </Container>
+  )}
+  {...props}
+/>;
+```
+
+<RenderItem data={sampleData} />
```

Comment:
How about renaming this and the file name to `RenderFunction`?
`RenderItem` looks like an actual component that should be used.

Furthermore, I think adding prefixes to example components would be more clear. For example,
```
ExRenderFunction
ExToggleElement
```

Reply:
Thanks for review~! I agree with your comment.